src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -579,7 +579,7 @@ export function getEventContent(event) {
       };
     case EVENT_TYPES.certified:
       return {
-        title: 'The Board received your appeal',
+        title: 'Your appeal was transferred to the Board of Veterans’ Appeals',
```

Comment:
Gina, can we change "transferred" to "sent" here?

Reply:
Yes sent works here.